.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
can you remove all those `.idea` files? maybe add the directory to the `gitignore`